setup.py

```diff
@@ -90,6 +90,8 @@ def run(self):
     "twine==2.0.0",  # Apache License 2.0
 ]
 
+examples_require = ["tensorflow==1.14.0"]  # Apache 2.0
```

Comment:
We're in the process of refactoring the `xain` package in a way that will remove the need for `tensorflow` in `install_requires`, so I'd recommend to keep the redundant require for a while - other examples will require other ML frameworks and it would be helpful to have those dependencies already separated from the `xain` ones